parse.py

```diff
@@ -48,33 +52,51 @@ def ocr_space_url(url, overlay=False, api_key= ocr_key, language='eng', isTable=
     :param isTable: parses output as a table
     """
 
-    payload = {'url': url,
-               'isOverlayRequired': overlay,
-               'apikey': api_key,
-               'language': language,
-               'isTable': isTable,
-               }
-    r = requests.post('https://api.ocr.space/parse/image',
-                      data=payload,
-                      )
-    #return r.content.decode()
+    payload = {
+        "url": url,
+        "isOverlayRequired": overlay,
+        "apikey": api_key,
+        "language": language,
+        "isTable": isTable,
+    }
+    r = requests.post("https://api.ocr.space/parse/image", data=payload,)
+    # return r.content.decode()
+
+    # Change this return in source code to return JSON object, not string
     return r.json()
 
-data = ocr_space_url('https://ocr.space/Content/Images/receipt-ocr-original.jpg')
+#API response and obtaing "LineText"
+def parse():
+    words = []
+    data = ocr_space_url("https://ocr.space/Content/Images/receipt-ocr-original.jpg")
+    for i in data.get("ParsedResults")[0]["TextOverlay"]["Lines"]:
```

Comment:
I think it would be cleaner to declare `lines` as a separate variable then use a list comprehension for `words`.

```
lines = data.get("ParsedResults")[0]["TextOverlay"]["Lines"]
words = [line.get("LineText") for line in lines]

Reply:
agreed, learned how to python 1 line for loops